crates/database/src/repositories/model.rs

```diff
@@ -140,16 +143,35 @@ impl ModelRepository {
             .query(
                 r#"
                 SELECT 
-                    id, model_name, model_display_name, model_description, model_icon,
-                    input_cost_per_token, output_cost_per_token,
-                    context_length, verifiable, is_active, created_at, updated_at
-                FROM models 
-                WHERE model_name = $1 AND is_active = true
+                    m.id, m.model_name, m.model_display_name, m.model_description, m.model_icon,
```

Comment:
This function is supposed to only get model from its canonical name, not from aliases. What about using this function? https://github.com/nearai/cloud-api/blob/0d59aa6a7115e6b0985583610e0cffc63169742f/crates/database/src/repositories/model.rs#L500